internal/errs/errs.go

```diff
@@ -36,6 +36,19 @@ type TransientError interface {
 	IsTransient()
 }
 
+type UserFacingError struct {
+	wrapped error
+	message string
+}
+
+func (e UserFacingError) Error() string {
+	return JoinMessage(Wrap(e.wrapped, e.message))
```

Comment:
Joining errors is the responsibility of the code that interprets them.

I suggest instead just using "User Facing Error: <message>" so it's clear from reading the standard error stack what we're dealing with.

Reply:
Is it possible to make this a code suggestion? I don't understand what you're asking for.